Core/EasylistStore.swift

```diff
@@ -34,16 +34,25 @@ class EasylistStore {
 
     }
 
-    var hasData: Bool {
-        return easylist != "" && easylistPrivacy != ""
+    var hasData = false
```

Comment:
One small buglet here. This variable has slowly taken on two different meanings
1. That there is data from the server
2. That processed data from the js layer was received

This means if I kill the first easylist request (using charles) and then try and reload via the content blocking reload button, it doesn't seem to work. hasData continues to return false until the js response is received.

Reply:
Gah, I was trying to avoid load those files in to memory is all.  I'll check against their presence on the file system instead.